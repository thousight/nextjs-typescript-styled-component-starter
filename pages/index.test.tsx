import { render, screen } from '@testing-library/react'
import Home from './index'

describe('Home', () => {
  it('matches snapshot', () => {
    const { container } = render(<Home />)

    expect(container).toMatchSnapshot()
  })

  it('renders app name with version', () => {
    render(<Home />)

    const appNameText = screen.getByText(/at version/i)

    expect(appNameText).toBeInTheDocument()
  })
})
