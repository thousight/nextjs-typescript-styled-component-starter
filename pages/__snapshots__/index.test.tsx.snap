// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home matches snapshot 1`] = `
<div>
  <main>
    <p
      class="sc-bdvvtL iQBoIu"
    >
      nextjs-typescript-styled-component-starter
       at version 
      0.1.0
    </p>
  </main>
</div>
`;
